packages/fhir-converter/src/templates/cda/Resources/Observation.hbs

```diff
@@ -68,11 +68,11 @@
         {{else if observationEntry.value.value}}
             "valueQuantity":
             {
-                "value":{{observationEntry.value.value}},
+                "value":"{{observationEntry.value.value}}",
                 "unit":"{{observationEntry.value.unit}}",
             }, 
         {{else}}
-            "valueString":"{{observationEntry.value._}}",
+            "valueString":"{{{parseReferenceData observationEntry.value._}}}",
```

Comment:
this patches this error: https://metriport.slack.com/archives/C06CQC7GQG3/p1706031364826159?thread_ts=1706027906.239089&cid=C06CQC7GQG3